Wiki Reorg: Add Constants and Variables Page (Learn Section) (#6135)
w3f/polkadot-wiki: docs/general/constants-variables.md

#### Staking Miner Max Submissions

The maximum number of submission for a staking miner on Polkadot is <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedMaxSubmissions" defaultValue={16}/>.

#### Total Issuance

Polkadot's total issuance is  <RPC network="polkadot" path="query.balances.totalIssuance" defaultValue="14883815224560918110" filter= "humanReadable"/> in the era <RPC network="polkadot" path="query.staking.currentEra" defaultValue="1553"/>.

#### Treasury Burn Factor

At the end of every spending period on Polkadot, <RPC network="polkadot" path="consts.treasury.burn" defaultValue={10000} filter="permillToPercent"/>% of the available funds are burned.

#### Treasury Spending Period

The spending period on Polkadot is currently <RPC network="polkadot" path="consts.treasury.spendPeriod" defaultValue={345600} filter="blocksToDays"/> days.

#### Unbonding Duration

The unbonding duration on Polkadot is set to <RPC network="polkadot" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> days.

</TabItem>
<TabItem value="kusama">

> :::info What to do with my KSM

Reviewer: ```suggestion
:::info What to do with KSM
```

#### Existential Deposit

The minimum number of tokens to keep an account alive on the Kusama Asset Hub is <RPC network="statemine" path="consts.balances.existentialDeposit" defaultValue={1000000000} filter="humanReadable"/>.

</TabItem>
<TabItem value="pp">

#### Identity Deposit

The creation of a Polkadot identity required a deposit of <RPC network="polkadotpeople" path="consts.identity.basicDeposit" defaultValue={2001700000} filter="humanReadable"/> and <RPC network="polkadotpeople" path="consts.identity.byteDeposit" defaultValue={100000} filter="humanReadable"/> (per byte) per each field beyond the legal name.

#### Sub-identity Deposit

The creation of a Polkadot sub-identity required a deposit of <RPC network="polkadotpeople" path="consts.identity.subAccountDeposit" defaultValue={2005300000} filter="humanReadable"/>.

</TabItem>
<TabItem value="kp">

#### Identity Deposit

> The creation of a Polkadot identity required a deposit of <RPC network="kusamapeople" path="consts.identity.basicDeposit" defaultValue={6672333321} filter="humanReadable"/> and <RPC network="kusamapeople" path="consts.identity.byteDeposit" defaultValue={333333} filter="humanReadable"/> (per byte) per each field beyond the legal name.

Reviewer: ```suggestion
The creation of a Kusama identity required a deposit of <RPC network="kusamapeople" path="consts.identity.basicDeposit" defaultValue={6672333321} filter="humanReadable"/> and <RPC network="kusamapeople" path="consts.identity.byteDeposit" defaultValue={333333} filter="humanReadable"/> (per byte) per each field beyond the legal name.
```

#### Sub-identity Deposit

> The creation of a Polkadot sub-identity required a deposit of <RPC network="kusamapeople" path="consts.identity.subAccountDeposit" defaultValue={6684333309} filter="humanReadable"/>.

Reviewer: ```suggestion
The creation of a Kusama sub-identity required a deposit of <RPC network="kusamapeople" path="consts.identity.subAccountDeposit" defaultValue={6684333309} filter="humanReadable"/>.
```

---

> id: constants-variables

Reviewer: **variables** is a generic mathematical term not often used in the blockchain context. It also has a different meaning in programming context when compared to what is listed on this page. Every item on this page is a queryable chain constant or a storage value, so I suggest to rename to `chain-constants-storage`
 
 We can also use `chain-state-values` or something along those lines.
 

```suggestion
id: chain-constants-storage
```

> title: Constants and Variables

Reviewer: ```suggestion
title: Chain Constants and Storage Values
```

> sidebar_label: Constants and Variables

Reviewer: We can find a better sidebar label. 

But **variables** is a generic mathematical term. Every item on this page is a queryable chain constant or a storage value. "Chain State Values" could be a shorter version of this title.

```suggestion
sidebar_label: Chain Constants and Storage Values
```

> description: Constants and Variables.

Reviewer: ```suggestion
description: Chain constants and storage values that can be queried from the live network nodes of Polkadot, Kusama and their system chains.
```

> keywords: [polkadot, kusama, constants, variables]

Reviewer: ```suggestion
keywords: [polkadot, kusama, constants, storage, values]
```

slug: ../constants-variables
---

import RPC from "./../../components/RPC-Connection"; import Tabs from "@theme/Tabs"; import TabItem
from "@theme/TabItem";

<!-- prettier-ignore-start -->
<Tabs groupId="chains" values={[ {label: 'Polkadot', value: 'polkadot'}, { label: 'Kusama', value: 'kusama'}, {label: 'Polkadot Asset Hub', value: 'ahp'}, {label: 'Kusama Asset Hub', value: 'ahk'}, {label: 'Polkadot People', value: 'pp'}, {label: 'Kusama People', value: 'kp'} ]}>

<TabItem value="polkadot">

> :::info What to do with my DOT

Reviewer: ```suggestion
:::info What to do with DOT
```

#### Child Bounty Payout Delay

The waiting time before claiming a Polkadot child bounty reward is <RPC network="polkadot" path="consts.bounties.bountyDepositPayoutDelay" defaultValue={115200} filter="blocksToDays"/> days.

#### Conviction Voting Lock Period

One conviction voting lock period on Polkadot equals <RPC network="polkadot" path="consts.convictionVoting.voteLockingPeriod" defaultValue={100800} filter="blocksToDays"/> days.

#### Existential Deposit

The minimum number of tokens to keep an account alive on the Polkadot Relay Chain is <RPC network="polkadot" path="consts.balances.existentialDeposit" defaultValue={333000000} filter="humanReadable"/>.

#### Inactive Issuance

Polkadot's inactive issuance is <RPC network="polkadot" path="query.balances.inactiveIssuance" defaultValue="1784854324418488473" filter= "humanReadable"/> in the era <RPC network="polkadot" path="query.staking.currentEra" defaultValue="1200"/>.

#### Index Deposit

The deposit to reserve an index on Polkadot is <RPC network="polkadot" path="consts.indices.deposit" defaultValue={100000000000} filter="humanReadable"/>.

#### Maximum Number of Nominators

> The maximum number of nominators on Polkadot is uncapped and currently sits at <RPC network="polkadot" path="query.staking.counterForNominators" defaultValue={50000}/>.

Reviewer: ```suggestion
The maximum number of nominators on Polkadot is uncapped and the current value is <RPC network="polkadot" path="query.staking.counterForNominators" defaultValue={50000}/>.
```

Author: implemented

#### Maximum Number of Proxies per Account

The maximum number of proxies per Polkadot account is <RPC network="polkadot" path="consts.proxy.maxProxies" defaultValue={32}/>. You can have the same proxy for multiple accounts.

#### Maximum Votes per Nominator

A nominator on Polkadot can select up to <RPC network="polkadot" path="consts.electionProviderMultiPhase.minerMaxVotesPerVoter" defaultValue={16} /> validators.

#### Minimum Active Bond

> The minimum amount of tokens to nominate on Polkadot is <RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>.

Reviewer: ```suggestion
The minimum amount of DOT required to nominate on Polkadot is <RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>.
```

Author: DOT already specified at the end of the sentence

<Tabs groupId="clients" values={[ {label: 'Polkadot', value: 'polkadot'}, { label: 'Kusama', value:
'kusama'} ]}>

<TabItem value="polkadot">

#### Treasury Spending Period

The spending period on Polkadot is currently <RPC network="polkadot" path="consts.treasury.spendPeriod" defaultValue={345600} filter="blocksToDays"/> days.

#### Minimum Bond to Participate in Staking

The minimum bond to nominate on Polkadot is <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={2500000000000} filter="humanReadable"/> while the minimum amount to join a pool is <RPC network="polkadot" path="query.nominationPools.minJoinBond" defaultValue={10000000000} filter="humanReadable"/>.

#### Existential Deposit

The minimum number of tokens to keep an account alive on the blockchain is <RPC network="polkadot" path="consts.balances.existentialDeposit" defaultValue={333000000} filter="humanReadable"/>.

#### Minimum Active Bond

The minimum amount of tokens to nominate is <RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>.

</TabItem>
<TabItem value="kusama">

#### Treasury Spending Period

> The spending period on Polkadot is currently <RPC network="kusama" path="consts.treasury.spendPeriod" defaultValue={86400} filter="blocksToDays"/> days.

Reviewer 2: ```suggestion
The spending period on Kusama is currently <RPC network="kusama" path="consts.treasury.spendPeriod" defaultValue={86400} filter="blocksToDays"/> days.
```

#### Minimum Bond to Participate in Staking

> The minimum bond to nominate on Polkadot is <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> while the minimum amount to join a pool is <RPC network="kusama" path="query.nominationPools.minJoinBond" defaultValue={1667000000} filter="humanReadable"/>.

Reviewer 2: ```suggestion
The minimum bond to nominate on Kusama is <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> while the minimum amount to join a pool is <RPC network="kusama" path="query.nominationPools.minJoinBond" defaultValue={1667000000} filter="humanReadable"/>.
```
